feathr_project/feathr/spark_provider/_localspark_submission.py

```diff
@@ -205,6 +209,11 @@ def wait_for_completion(self, timeout_seconds: Optional[float] = 500) -> bool:
                 contents = f.read()
                 logger.error(contents)
             return False
+        elif proc.returncode == 143:
+            logger.info(
+                f"Spark job with pid {self.latest_spark_proc.pid} finished in: {int(job_duration)} seconds."
```

Comment:
Better include some hint about 143 returncode in log or add some comment here. 
AFAIK, when local spark terminate the job due to no response for certain seconds, it will return 143. However, there are still other cases that 143 is returned not by design.

Reply:
good point, added